src/pages/resultsView/enrichments/MutationEnrichmentsTab.tsx

```diff
@@ -21,44 +22,48 @@ export default class MutationEnrichmentsTab extends React.Component<IMutationEnr
     private uniqueColorGetter = makeUniqueColorGetter();
 
     @autobind
-    private onProfileChange(molecularProfile: MolecularProfile) {
-        this.props.store._selectedEnrichmentMutationProfile = molecularProfile;
+    private onProfileChange(profileMap:{[id:string]:MolecularProfile}) {
+        this.props.store.setMutationEnrichmentProfileMap(profileMap);
     }
 
-    public render() {
-        if (this.props.store.mutationEnrichmentData.isPending) {
-            return <Loader isLoading={true} center={true} size={"big"}/>;
-        } else if (this.props.store.mutationEnrichmentData.isError) {
-            return <ErrorMessage/>;
-        } else {
+    readonly tabUI = MakeMobxView({
+        await: () => [this.props.store.studies,  this.props.store.mutationEnrichmentData, this.props.store.selectedMutationEnrichmentProfileMap],
+        render: () => (
+            <div data-test="MutationEnrichmentsTab">
+                <EnrichmentsDataSetDropdown
+                    dataSets={this.props.store.mutationEnrichmentProfiles}
+                    onChange={this.onProfileChange}
+                    selectedProfileByStudyId={this.props.store.selectedMutationEnrichmentProfileMap.result!}
+                    molecularProfileIdToProfiledSampleCount={this.props.store.molecularProfileIdToProfiledSampleCount}
+                    studies={this.props.store.studies.result!}
+                />
+                <AlterationEnrichmentContainer data={this.props.store.mutationEnrichmentData.result!}
+                    headerName={"Mutations"}
```

Comment:
just confirming you want to show "Mutations" even if theres only one mutation profile?